### ISSUE TITLE
MetaProxy

### ISSUE THREAD

**Issue by davidbios** [metahashorg/metagate#19]
Any fix for MetaProxy is not running
Checking servers and establishing Forging conenctions can take up to 5 minutes. Please be patient. ?

**Comment by briancfalls** [comment]
MetaProxy is not running, did not get paid or reset at 00:00UTC since update to 1.20.2, my other computer I did not update and is still working, Telegram says only us still that did not update yet are still forging.

**Comment by ijanssen** [comment]
Please open 'Settings' and set 'Enable forging on this machine'. If it's on please reset it (off and on later).

**Comment by andreafno** [comment]
Me too. In "Settings" I can only see language selection.

**Comment by ijanssen** [comment]
> Me too. In "Settings" I can only see language selection.
You need to login. This setting is not available without login.

**Comment by andreafno** [comment]
![Schermata 2019-12-21 alle 10 35 23](https://user-images.githubusercontent.com/9588996/71306173-a74ff080-23dd-11ea-9267-f8866cbdef56.png)

**Comment by ijanssen** [comment]
Yes, create account and login on first page after start (app://Login). You will have another settings.

**Comment by andreafno** [comment]
Ok, thank you. Now I see "Enable forging on this machine", but I always see "Metaproxy is not running". Let's see if tomorrow something changes.

**Comment by supervolot** [comment]
thx ijanssen that worked fine (mac and W10)

**Comment by Sebescen81** [comment]
My place is the same. MetaProxy is not running. Already put the hook in and out. Still the same.

**Comment by a2en** [comment]
I had this same issue. This is how I got it working. I first turned of forging on settings and logged out. Then turned on again after logging in.
